lens/registry.go

```diff
@@ -98,9 +101,53 @@ func (r *lensRegistry) SetMigration(ctx context.Context, txn datastore.Txn, cfg
 }
 
 func (r *lensRegistry) cacheLens(txn datastore.Txn, cfg client.LensConfig) error {
-	locker, lockerAlreadyExists := r.lensPoolsBySchemaVersionID[cfg.SourceSchemaVersionID]
-	if !lockerAlreadyExists {
-		locker = r.newPool(r.poolSize, cfg)
+	inversedModuleCfgs := make([]model.LensModule, len(cfg.Lenses))
+	for i, moduleCfg := range cfg.Lenses {
+		// Reverse the order of the lenses for the inverse migration.
+		inversedModuleCfgs[len(cfg.Lenses)-i-1] = model.LensModule{
```

Comment:
question: If `len(cfg.Lenses) == 1` then wouldn't this be `inversedModuleCfgs[-1]` for first index (`i=0`)?

Reply:
:) I don't think so:

`len(cfg.Lenses) = 1`
`i = 0`
Results in:
`inversedModuleCfgs[1-0-1]`

Reply:
I needed sleep, rip haha